assets/css/modules/_landing.scss

```diff
@@ -61,6 +61,11 @@
   .play-store-badge {
     height: 60px;
   }
+  @media (max-width: 992px) {
```

Comment:
I think this should be `max-width: 991px` https://v4-alpha.getbootstrap.com/layout/overview/#responsive-breakpoints